CONTRIBUTING.md

```diff
@@ -1,149 +1,235 @@
 # CONTRIBUTING
 
-## Contributing to documentation
+The Ruby LSP repository has three sub projects inside:
 
-Ruby LSP uses [Jekyll](https://jekyllrb.com/) to generate the documentation, whose source lives under the `/jekyll` folder.
+- The `ruby-lsp` gem, which is the language server implementation. This is the top level of the repository. Most features are implemented here since everything implemented in the server is available to all editors
+- The VS Code extension, which exists under the `vscode` directory. Any custom VS Code features are implemented here
+- Its documentation website, which exists under the `jekyll` directory. All user facing documentation for both the Ruby LSP and the Rails addon is contained here
```

Comment:
```suggestion
- the **language server** (`ruby-lsp`), which exists at the top level of the repository. Most features are implemented here since everything implemented in the server is available to all editors
- the **VS Code extension**, which exists under the `vscode` directory. Any custom VS Code features are implemented here
- the **documentation** website, which exists under the `jekyll` directory. All user facing documentation for both the Ruby LSP and the Rails addon is contained here
```